log4j-core/src/main/java/org/apache/logging/log4j/core/config/builder/impl/BuiltConfiguration.java

```diff
@@ -131,8 +129,9 @@ public void createAdvertiser(final String advertiserString, final ConfigurationS
         super.createAdvertiser(advertiserString, configSource, buffer, contentType);
     }
 
+    @Override
     public StatusConfiguration getStatusConfiguration() {
-        return statusConfig;
+        return super.getStatusConfiguration();
     }
```

Comment:
Can't we simply remove this method definition here? Dynamic dispatching will call `super` anyway.